invenio_circulation/config.py

```diff
@@ -47,12 +47,16 @@
 Items that have attached loans with these circulation statuses are
 not available to be loaned by patrons."""
 
-CIRCULATION_STATES_LOAN_COMPLETED = ["ITEM_RETURNED"]
+CIRCULATION_STATES_LOAN_COMPLETED = ['ITEM_RETURNED']
```

Comment:
We are using [black](https://github.com/psf/black) for CDS (not yet Invenio) as a not opinionated code formatter (with the exception of line length and imports sorting) to avoid such changes, in case you want to have a look.

Reply:
Thanks, I will have a look!